recipes-nodes/render-config/render-config.bb

```diff
@@ -46,4 +46,4 @@ RDEPENDS:${PN} += "jq curl coreutils mustache"
 inherit update-rc.d
 
 INITSCRIPT_NAME = "fetch-config"
-INITSCRIPT_PARAMS = "defaults 90"
+INITSCRIPT_PARAMS = "defaults 80"
```

Comment:
Start before System API service since it expects configuration from BuilderHub

Reply:
I thought it should start after the CVM-reverse-proxies. 
Here you are starting it before them. should we adjust? If yes, then this should be 88.
Namely, date-sync, cvm-proxy-server, cvm-proxy-client, system-api, rest of services ...

Reply:
cvm-proxy -> fetch-config -> system-api